lib/marked.js

```diff
@@ -944,7 +949,7 @@ Renderer.prototype.tablerow = function(content) {
 Renderer.prototype.tablecell = function(content, flags) {
   var type = flags.header ? 'th' : 'td';
   var tag = flags.align
-    ? '<' + type + ' style="text-align:' + flags.align + '">'
+    ? '<' + type + ' align="' + flags.align + '">'
```

Comment:
Re: 
* https://www.w3schools.com/tags/att_table_align.asp ... Not HTML5 compliant according to them. Also in a few other places in this PR. Is that compliance being discarded?

Reply:
This is GFM compliant, not HTML  compliant.  Compliance with GFM requires that we break HTML compliance.  

https://github.github.com/gfm/#example-192

Reply:
@tomtheisen 
> This is GFM compliant, not HTML compliant. Compliance with GFM requires that we break HTML compliance.

That's why it's in the form of a question. Good to know of a major breaking change as @styfle put it in the issue. If we continue to use this package we'll need to undo this for our sanitizer *(which we do for HTML5 anyhow)* as we are always striving for HTML5 compliance. Anyone using this change that has a HTML5 compliant badge will need to do this as well since it's non-standard or they'll lose their logo/rating.

Since I've been on GH just about as long as they've been around I can make an *educated guess* that they did this spec for older browser compatibility and also their sanitizer. So when the HTML4.x specifications drop out *(presumably around 2020ish when everything hits the fan)* the backward compatibility reintroduced here will not render tables correctly and it may need to be reversed. Time will tell.